scripts/01_fetch_messages.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -ne 6 ]; then
+    echo "Usage: sh 01_fetch_messages.sh <user> <rapid-pro-root> <rapid-pro-server> <rapid-pro-token> <data-root> <test-contacts-path>"
+    echo "Downloads radio show answers from each show"
+    exit
+fi
+
+USER=$1
+RP_DIR=$2
+RP_SERVER=$3
+RP_TOKEN=$4
+DATA_ROOT=$5
+TEST_CONTACTS_PATH=$6
+
+cd "$RP_DIR/fetch_runs"
+
+mkdir -p "$DATA_ROOT/01 Raw Messages"
+
+SHOWS=(
+    "emergency_maap_pdm1_survey"
+    "emergency_maap_pdm2_survey"
+    "emergency_maap_pdm3_survey"
+    "emergency_maap_pdm4_survey"
+    "emergency_maap_pdm5_survey"
+    "emergency_maap_demogs"
```

Comment:
Why are you retrieving the demogs in 'all' mode rather than 'latest-only'?

Reply:
Because I'm yet to run this part of the pipeline, this is for the first run